js/views/pageNavVw.js

```diff
@@ -462,13 +461,34 @@ module.exports = baseVw.extend({
     this.currentWindow.reload();
   },
 
+  trimAddressBar: function() {
+    this.addressInput.val(function (i, value) {
+      return value.replace('ob://', '');
+    });
+  },
+
+  untrimAddressBar: function(){
+    this.addressInput.val(function (i, value) {
+      value = value.startsWith('ob://') ? value : 'ob://' + value;
+      return value;
+    });
+  },
+
```

Comment:
The functions to add or remove the prefix to the address bar have been consolidated above.